@navikt/core/css/stepper.css

```diff
@@ -1,99 +1,180 @@
 .navds-stepper {
-  display: flex;
-  flex-direction: column;
-  padding: 0;
-  margin: 0;
+  --navds-stepper-circle-size: 1.75rem;
+  --navds-stepper-border-width: 2px;
+  --navds-stepper-line-length: 1rem;
+
+  color: var(--navds-semantic-color-interaction-primary);
+  font-weight: 600;
+  line-height: 1;
   list-style: none;
+  padding-left: 0;
+  margin: 0;
 }
 
-/* Step */
-.navds-stepper__step {
-  position: relative;
-  cursor: pointer;
-  padding: calc(2.25rem / 2) 0;
+.navds-stepper--horizontal {
   display: flex;
-  background: none;
-  border: none;
-  appearance: none;
-  width: 100%;
-  color: var(--navds-semantic-color-text);
+  align-items: start;
+  text-align: center;
+}
+
+.navds-stepper__item {
```

Comment:
Stepindicator har satt `min-width: 10rem` på horisontalt item. Har det vært en ide å beholde for stepper?

Før
<img width="155" alt="Screenshot 2022-06-27 at 14 54 43" src="https://user-images.githubusercontent.com/26967723/175946673-21d18922-f89c-4487-a9c7-f0806fb0313a.png">

Etter
<img width="231" alt="Screenshot 2022-06-27 at 14 54 56" src="https://user-images.githubusercontent.com/26967723/175946676-468481e5-f6a1-48ea-ae08-a926818683b1.png">

Reply:
Ja, det kan godt tenkes vi bør ha noe sånt. Her er mitt forslag:

```css
.navds-stepper__content { 
  max-width: 24ch 
}
```